libweasyl/libweasyl/models/tables.py

```diff
@@ -664,6 +664,21 @@ def default_fkey(*args, **kwargs):
 Index('ind_searchmapsubmit_targetid', searchmapsubmit.c.targetid)
 
 
+searchmapartist = Table(
+    'searchmapartist', metadata,
+    Column('tagid', Integer(), primary_key=True, nullable=False),
+    Column('targetid', Integer(), primary_key=True, nullable=False),
+    Column('settings', CharSettingsColumn({
```

Comment:
Maybe these should be separate tables instead? `artist_preferred_tags` and `artist_non_preferred_tags`, or something like that.